releases/release-1.31/release-notes/maps/pr-121508-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 121508
+releasenote:
+  text: |-
+    kube-scheduler implemented scheduling hints for the CSILimit plugin.
+    The scheduling hints allow the scheduler to retry scheduling a Pod that was previously rejected by the CSILimit plugin if a deleted pod has a PVC from the same driver.
```

Comment:
Can we define PersistantVolumeClaim (PVC) in addition to having the abbreviation here?